src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = [...moviesFromServer].filter(movie => {
```

Comment:
```suggestion
  const visibleMovies = moviesFromServer.filter(movie => {
```
filter повертає новий масив завжди, не має потреби створювати ще одну копію